### ISSUE TITLE
getaccountaddress No longer compatible with Bitcoin/OpenTrade

### ISSUE THREAD

**Issue by daygle** [3s3s/opentrade#159]
After upgrading the Bitcoin wallet  I have noticed that OpenTrade is no longer able to generate a wallet address for Bitcoin only. Other coins are ok.

Further investigation shows that the 'getaccountaddress' has been removed from the Bitcoin wallet and replaced with getnewaddress?

https://github.com/bitcoin-core/bitcoin-devwiki/wiki/0.17.0-Release-notes

Looks like 'move' has been removed too, I think OpenTrade uses move?

I urgently need a fix for this as I am unable to use a previous Bitcoin version as my wallet has been upgraded.

Does anyone have a fix for this?

**Comment by x3c** [comment]
For what it's worth, you can always revert to a previous bitcoin version.

- dump your wallet with the dumpwallet <filename> command
- downgrade to previous version, remove wallet.dat (backups etc!)
- import the dumped wallet with the importwallet <filename> command
- remove the dump filename, which is dumped in clear text and unprotected by any password.

Hope it helps.

**Comment by Thedabest** [comment]
if the update 3s3s up command rpc 0.17.0 and still still old command, easy easy add more ethereum and monero, equivalent :)

**Comment by arvoim** [comment]
0.17 can still be used, those rpc calls will be removed from 0.18

see: ['label' and 'account' APIs for wallet](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.17.0.md#label-and-account-apis-for-wallet)

TLDR; start bitcoind with '-deprecatedrpc=accounts'

**Comment by Puual** [comment]
You can edit your bitcoin.conf and insert deprecatedrpc=accounts

Then you don't have to think about it in case you restart your wallet.

**Comment by daygle** [comment]
@arvoim @Puual I think you will find although you add deprecatedrpc=accounts it will still fail. But have a go yourself, don't say I didn't warn you.

I beleive the 'move' function was removed, this is something that opentrade heavily relies on.

**Comment by Puual** [comment]
@daygle 
That shocked me for a moment. Tested it with Bitcoin on testnet.

- Deposit to user1
- exchanged to user2
- withdraw from user2 back to the faucet 
- withdraw from user1 to deposit it at user2

Everything worked as it should be

Might there be a thing that should also be tested?

**But I agree, this is something _serious_. Everyone should test this for himself**
